Shakefile.hs

```diff
@@ -121,7 +125,8 @@ tex2pdf = do
     need [ maintexfile ]
 
     -- read the Agda inputs from the tex file
-    agdainputs <- getAgdaInputs <$> readFileLines maintexfile
+    raw <- liftIO $ BS.readFile maintexfile
+    let agdainputs = getAgdaInputs (map Text.unpack (Text.lines (TE.decodeUtf8 raw)))
```

Comment:
Added a commit which uses a different approach for UTF8 handling that (externally) doesn't go through bytestring